.github/workflows/test-coverage.yaml

```diff
@@ -28,6 +28,7 @@ jobs:
 
       - name: test-coverage
         run : |
+          remotes::install_version("mlr3viz", version = "0.5.10")
```

Comment:
it seems works in workflows, but we shoud add `remotes` installation in Dockerfile as well, or use `devtools::install_deps()` instead before installing `mlr3viz`